src/test/integration/configuration_example.json

```diff
@@ -1,5 +1,6 @@
 {
     "hostname": "https://api.example.com/api/v1/locations",
+    //https://docker.for.mac.localhost:8080/api/v1/locations
```

Comment:
Could you put this link in the readme instead since json can't have comments?

Reply:
Fixed.